pandas/tests/extension/base/io.py

```diff
@@ -11,6 +11,8 @@
 class BaseParsingTests(BaseExtensionTests):
     @pytest.mark.parametrize("engine", ["c", "python"])
     def test_EA_types(self, engine, data):
+        if data.dtype.kind == "c":
```

Comment:
Same here

Reply:
Done.  This led to lots of interface changes to get `request` everwhere it needed to be.